wasm/build-wasm.sh

```diff
@@ -1,10 +1,7 @@
 # TODO: Ensure that version is up to date
 cp README.md wasm/
 cp LICENSE.md wasm/
-npx wasm-pack@0.10.3 build --target web --out-dir wasm/stylua.web -- --features lua52,lua53,lua54,luau
-
-# workaround for bundler usage
-echo "export { getImports as __getImports, finalizeInit as __finalizeInit }" >> wasm/stylua.web/stylua_lib.js
```

Comment:
How come we are removing this line?

Reply:
Good question, now that you mention this I've realized that other files in the parent directory seem to depend on these exports. The issue is the file which is generated by the latest version of wasm-pack doesn't contain `getImports` or `finalizeInit`. I'm not exactly clear on what these were actually for though...

So I've converted this to a draft for now. I'm kinda busy right now so it might be a little bit before I can dig into this and figure out what those were for.